.github/workflows/dependabot-alerts.yml

```diff
@@ -1,10 +1,8 @@
 name: 'Dependabot Alerts Action'
 
 on:
-  workflow_dispatch:
```

Comment:
I would preserve the alternate triggers - what do you think?

Reply:
Ah good point, done!